index.html

```diff
@@ -1127,6 +1163,150 @@ <h2 property="schema:name">Discovering a complete Solid profile</h2>
             </div>
           </section>
 
+          <section id="reading-writing-profiles" inlist="" rel="schema:hasPart" resource="#reading-writing-profiles">
+            <h2 property="schema:name">Reading and Writing Profiles</h2>
+            <div datatype="rdf:HTML" property="schema:description">
+                <section id="reading-profile" inlist="" rel="schema:hasPart" resource="#reading-profile">
+                    <h3 property="schema:name">Reading Profile</h3>
+                    <div datatype="rdf:HTML" property="schema:description">
+                        <p about="" id="reader-application-webid-profile" rel="spec:requirement"
+                            resource="reader-application-webid-profile">
+                            <span property="spec:statement">To retrieve a representation of a Solid WebID Profile, <a
+                                    rel="spec:requirementSubject" href="#ReaderApplication">Reader Application</a> <span
+                                    rel="spec:requirementLevel" resource="spec:MUST">MUST</span> MUST make
+                                an <code>HTTP GET</code> request targeting a Solid WebID Profile resource including a
+                                <code>Content-Type header</code> requesting a representation in <code>text/turtle</code> or
+                                <code>application/ld+json.</code></span>
+                        </p>
+                        <section id="webid-profile-data-model" inlist="" rel="schema:hasPart"
+                            resource="#webid-profile-data-model">
+                            <h3 property="schema:name" content="WebID Profile Data Model">Solid WebID Profile</h3>
+                            <div datatype="rdf:HTML" property="schema:description">
+                                <p>The <a href="#solid-profile">Solid WebID Profile</a> has the following properties:</p>
+                                <ul>
+                                    <li>One <code>rdf:type</code> property whose object is <code>foaf:Agent</code>.</li>
+                                    <li>One <code>space:preferencesFile</code>.</li>
```

Comment:
The namespaces section uses `pim` as a prefix, at the same time this line and the rest of the section use `space` as a prefix.

Reply:
* N3 and Turtle serializations of http://www.w3.org/ns/pim/space use `ws` as the prefix for `http://www.w3.org/ns/pim/space#`
* http://prefix.cc/pim has an entry for the above, and http://prefix.cc/space has an entry for something entirely different.
* https://github.com/solid/solid-namespace/blob/7498b4d93b543c1c1508d2542013b97bd10db127/index.js#L51 uses `space`
* Not that this matters much but the source file is [space.n3](https://github.com/solid/vocab/blob/main/space.n3).
* Not that this matters much but I've been using `pim` although I do find `ws` or `space` relatively more clear because there are other vocabs under `pim/`.

Pick one at random :)

Reply:
Let's stick to `pim` for now. We're still in drafts land... can come back to this and other prefixes across all specs if we need to.